### PR TITLE
Update home.html text

### PR DESCRIPTION
**Pull request by libcce**
CE edit to home text

### CODE REVIEW

File: _layouts/home.html [hunk 1]
```diff
@@ -26,9 +26,11 @@ <h1>{{ site.title }}</h1>
...
+        <p>The ModelShare Programme stems from the work of the Open Modeling Foundation and the Certification Working Group. The Open Modeling Foundation is an alliance of modeling organizations that coordinates and administers a common, community developed body of standards and best practices among diverse communities of modeling scientists. The Certification Working Group (CWG) develops and administers programs to certify model code that meets OMF standards for modeling best practices, and officially recognize model developers who apply these standards.</p> 
+
+<p></p>ModelShare will convene experts in a series of online workshops in early to mid 2024 looking at current and future approaches to sharing models throughout the research lifecycle. Through these sessions, we expect to gather community input to inform best practices and recommendations for improving the sharing of models and ultimately aiding the quality and reproducibility of research outcomes.</p>
```
Comment (Ismael-KG) [inline _layouts/home.html]: Remove `</p>` immediately after `<p>`

File: _layouts/home.html [hunk 2]
```diff
@@ -26,9 +26,11 @@ <h1>{{ site.title }}</h1>
...
+        <p>The ModelShare Programme stems from the work of the Open Modeling Foundation and the Certification Working Group. The Open Modeling Foundation is an alliance of modeling organizations that coordinates and administers a common, community developed body of standards and best practices among diverse communities of modeling scientists. The Certification Working Group (CWG) develops and administers programs to certify model code that meets OMF standards for modeling best practices, and officially recognize model developers who apply these standards.</p> 
+
+<p></p>ModelShare will convene experts in a series of online workshops in early to mid 2024 looking at current and future approaches to sharing models throughout the research lifecycle. Through these sessions, we expect to gather community input to inform best practices and recommendations for improving the sharing of models and ultimately aiding the quality and reproducibility of research outcomes.</p> 
+
+<p>You can stay informed by joining our Google Group here. Members of the Google Group will receive email updates about the ModelShare Programme.</p>
```
Comment (Ismael-KG) [inline _layouts/home.html]: Make "here" a link; `<a href="https://groups.google.com/g/omf-cwg-modelshare">here</a>`.

**Review by Ismael-KG — CHANGES_REQUESTED** [review]

**Comment by libcce** [comment]
FYI you can also accept the PR and make the changes. Might be the quickest path in the future.

**Review by Ismael-KG — APPROVED** [review]
Thank you!

**Pull request merged**